axes/models.py

```diff
@@ -51,6 +51,25 @@ class Meta:
         unique_together = [["username", "ip_address", "user_agent"]]
 
 
+class AccessAttemptExpiration(models.Model):
+    access_attempt = models.OneToOneField(
+        AccessAttempt,
+        primary_key=True,
+        on_delete=models.CASCADE,
+        related_name="expiration",
+        verbose_name=_("Access Attempt"),
+    )
+    expires_at = models.DateTimeField(
+        _("Expires At"),
+        null=True,
```

Comment:
These do not need to be null or blank as these rows should only ever be created in a case where the expirations are on, right?